<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="go_to_private_office">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Личный кабинет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_to_create_tests">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Переход к
 созданию тестов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_to_catalog">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Переход к
 каталогу тестов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutbtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выйти из аккаунта</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
